.circleci/config.yml

```diff
@@ -81,7 +81,7 @@ jobs:
   ui_test:
     <<: *container_config
     docker:
-      - image: circleci/node:7.10.0-browsers
+      - image: circleci/node:8.11.1-browsers
```

Comment:
maybe we should make these versions into anchors

Reply:
Good idea!